tests/base/time_stepping_01.output

```diff
@@ -27,89 +45,133 @@ DEAL::Cash-Karp
 DEAL::0
 DEAL::Forward Euler first order
 DEAL::convergence rate
-DEAL::0.496119
-DEAL::0.634657
-DEAL::0.763742
-DEAL::0.862050
-DEAL::0.924783
-DEAL::0.960617
-DEAL::0.979834
-DEAL::0.989794
-DEAL::0.994866
+DEAL::0.4961
+DEAL::0.6347
+DEAL::0.7637
+DEAL::0.8621
+DEAL::0.9248
+DEAL::0.9606
+DEAL::0.9798
+DEAL::0.9898
+DEAL::0.9949
 DEAL::Runge-Kutta third order
 DEAL::convergence rate
-DEAL::7.01207
-DEAL::0.0523936
-DEAL::2.00049
-DEAL::2.55207
-DEAL::2.78347
-DEAL::2.89332
-DEAL::2.94704
-DEAL::2.97361
-DEAL::2.98681
+DEAL::7.012
+DEAL::0.05239
+DEAL::2.000
+DEAL::2.552
+DEAL::2.783
+DEAL::2.893
+DEAL::2.947
+DEAL::2.974
+DEAL::2.987
 DEAL::Strong Stability Preserving Runge-Kutta third order
 DEAL::convergence rate
-DEAL::2.44463
-DEAL::2.72007
-DEAL::2.86894
-DEAL::2.93941
-DEAL::2.97135
-DEAL::2.98614
-DEAL::2.99319
-DEAL::2.99663
-DEAL::2.99832
+DEAL::2.445
+DEAL::2.720
+DEAL::2.869
+DEAL::2.939
+DEAL::2.971
+DEAL::2.986
+DEAL::2.993
+DEAL::2.997
+DEAL::2.998
 DEAL::Runge-Kutta fourth order
 DEAL::convergence rate
-DEAL::3.32883
-DEAL::3.64708
-DEAL::3.86926
-DEAL::3.95478
-DEAL::3.98399
-DEAL::3.99389
-DEAL::3.99745
-DEAL::3.99887
-DEAL::3.99923
+DEAL::3.329
+DEAL::3.647
+DEAL::3.869
+DEAL::3.955
+DEAL::3.984
+DEAL::3.994
+DEAL::3.997
+DEAL::3.999
+DEAL::3.999
+DEAL::Low-storage Runge-Kutta stage 3 order 3
+DEAL::convergence rate
+DEAL::2.412
+DEAL::2.660
+DEAL::2.829
+DEAL::2.917
+DEAL::2.960
+DEAL::2.980
+DEAL::2.990
+DEAL::2.995
+DEAL::2.998
+DEAL::Low-storage Runge-Kutta stage 5 order 4
+DEAL::convergence rate
+DEAL::3.065
+DEAL::3.525
+DEAL::3.779
+DEAL::3.896
+DEAL::3.950
+DEAL::3.975
+DEAL::3.988
+DEAL::3.994
+DEAL::3.994
+DEAL::Low-storage Runge-Kutta stage 7 order 4
+DEAL::convergence rate
+DEAL::8.762
+DEAL::-0.5791
+DEAL::3.296
+DEAL::3.716
+DEAL::3.871
+DEAL::3.938
+DEAL::3.969
+DEAL::3.987
+DEAL::4.099
+DEAL::Low-storage Runge-Kutta stage 9 order 5
+DEAL::convergence rate
+DEAL::4.173
+DEAL::4.669
+DEAL::4.854
+DEAL::4.933
+DEAL::4.968
+DEAL::4.984
+DEAL::4.996
+DEAL::5.060
+DEAL::3.170
```

Comment:
What is going on with the convergence rates here. Are we hitting machine precision problems? If yes, remove the last 1-2 refinements.